### PR TITLE
Updating with the Social venue info

### PR DESCRIPTION
**Pull request by mavi-sqr**
https://seqera.atlassian.net/browse/DP-546

### CODE REVIEW

**Comment by mavi-sqr** [comment]
Hi @ewels 
I saw there is a rule related to commits that didn't pass. Can we merge it anyway?

**Comment by ewels** [comment]
Weird, that test doesn't usually run on this repo 🤔 It's present on the Nextflow repo, wonder what brought it over here..

It's easy to fix - every commit needs to be signed. So can do the following:

```bash
git commit -s --amend
git push --force
```

The `-s` adds the signature suffix to the commit message.

But seeing as we've not had it until now on this repo I'm not sure that there's any point. I'll take a look in a bit and see if I can figure out why it's there / how to remove it again.

File: src/pages/2024/barcelona/travel.astro [hunk 1]
```diff
@@ -114,7 +114,38 @@ import SEO from "@components/SEO";
...
+          <div class="w-full md:w-1/2 p-2 min-h-[300px] relative">
+            <div class="h-full w-full relative rounded-lg overflow-hidden">
+              <MapEmbed
+                src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2993.456380010479!2d2.1707582999999997!3d41.385894199999996!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x12a4a2f137f6e727%3A0x4099c128fa4b2ab0!2sHard%20Rock%20Caf%C3%A8!5e0!3m2!1sen!2ses!4v1727953819228!5m2!1sen!2ses"
+              />
+            </div>
+          </div>
+          <div class="w-full md:w-1/2 p-2 min-h-[300px]">
+            <Box>
+              <p class="h3 mb-6">Hard Rock Cafè</p>
```
Comment (ewels) [inline src/pages/2024/barcelona/travel.astro:127]:
```suggestion
              <p class="h3 mb-6">Hard Rock Café</p>
```

File: src/pages/2024/barcelona/travel.astro [hunk 2]
```diff
@@ -114,7 +114,38 @@ import SEO from "@components/SEO";
...
+                  target="_blank"
+                  href="https://maps.app.goo.gl/7yUR2n3GoAYrjoNJA"
+                >
+                  <Icon name="mdi:map-marker" class="-mt-1" />
+                  Pl. de Catalunya, 21, Ciutat Vella, 08002 Barcelona
+                </a>
+              </p>
+              <p class="mb-3 font-thin text-sm">
+                Just a short 30-minute walk from the Hackathon at the World
+                Trade center, join us at the Hard Rock Cafè located on La
```
Comment (ewels) [inline src/pages/2024/barcelona/travel.astro:139]:
```suggestion
                Trade center, join us at the Hard Rock Café located on La
```

**Comment by ewels** [comment]
Ok it's a Catalan thing 😆  LGTM, merging!

**Pull request merged**